### DISCUSSION TITLE
Nesting parent dynamic size fit to children

### DISCUSSION THREAD

**Discussion by AlexanderYW** [bcakmakoglu/vue-flow#219] [Q&A] [answered]
Hi,

Is it possible to have parent shrink og expand to the size of the children's inside it and their positions?

So when ever a user is moving a child, parent either expand or decrease in size

**Comment by bcakmakoglu** [comment, marked as answer]
It is possible to expand the parent (using the `expandParent` property on your child node)
```js
const nodes = ref([
	  {
		  id: '1',
		  label: 'Node 1',
		  position: { x: 0, y: 0 }
	  },
	  {
		  id: '1a',
		  label: 'Node 1a',
		  position: { x: 0, y: 0 },
		  expandParent: true
	  }
])
```

Currently it is not possible, using a built-in mechanism, to collapse the same parent again.
That's something you would have to implement yourself (might be a feature in a future release but there's no plans yet).

**Reply by AlexanderYW** [reply]
Yes i could read about the expand parent, was just hoping there was something to shrink it again

Is there somewhere I could request this feature or this discussion enough? :)

**Reply by bcakmakoglu** [reply]
If you could open a new discussion requesting that specific feature, that would be great :)
There is actually the same request for react-flow so it seems like a wanted feature but currently I am prioritizing getting Vue Flow fully tested and released as `1.0.0` 😅

**Reply by mikila85** [reply]
still this feature not out? lib dead?

**Reply by bcakmakoglu** [reply]
No this feature simply has 0 priority.
If you want it, feel free to implement it and suggest an example for the docs or open a PR but it’s not going to land anytime soon since VueFlow 2.0 is priority atm and already is overdue.
As the sole maintainer of VueFlow my time is limited since I also work a regular job and have a private life as well.

**Comment by mikila85** [comment]
Thx for the comment, I'll try to find a way to do this properly and create
a PR
